git-filter-repo

```diff
@@ -3580,9 +3587,9 @@ class RepoFilter(object):
       print("        longer be related; consider re-pushing it elsewhere.")
     subproc.call('git remote rm origin'.split(), cwd=target_working_dir)
 
-  def _final_commands(self):
+  def _final_commands(self, callback_data):
```

Comment:
Same issue as above; you are passing an extra callback_data parameter, but then ignoring it and using self._callback_data.  What's the reason for passing it?  (Was this leftover from a previous attempting at passing things around and just didn't get cleaned up, by chance?)